test/autoInject.js

```diff
@@ -224,4 +224,33 @@ describe('autoInject', () => {
             done()
         })
     })
+
+        it('should not be subject to ReDoS', () => {
```

Comment:
Is this line intended too far? (I'm surprised if there isn't a lint rule for this.)